apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimSearch.json

```diff
@@ -103,7 +103,7 @@
                 }
               ]
             },
-            "system": "https://bluebutton.cms.gov/resources/variables/fi_doc_clm_cntrl_num",
+            "system": "https://bluebutton.cms.gov/resources/variables/fi_doc_clm_cntl_num",
```

Comment:
If this had been missing from v2 why was it in the v2 expected response? Were we just not checking this despite it being in the file?

Reply:
This is a Claim FHIR resource, not an EOB.

Reply:
These were missing from the Adjudicated resources, but we had added them for our Partially Adjudicated ones, though with a typo.  These are also obsolete anyways I believe, as a future PR (#972) will be changing them to PA specific ones